docs/source/api.rst

```diff
@@ -1,5 +1,84 @@
 AA Toolbox API
 ==============
 
-.. automodule:: aatoolbox
+Data sources
+------------
+
+CHIRPS
+^^^^^^
+
+.. automodule:: aatoolbox.datasources.chirps.chirps
```

Comment:
Do you know if it's possible to add the items from `aatoolbox` directly, i.e. instead of `aatoolbox.datasources.chirps.chirps`, just `aatoolbox.chirps`? I can't get it to work. My concern is now that the full import path shows up it could be a bit confusing. But it's not a big deal.